src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -2201,9 +2231,13 @@ Status ColorControlServer::moveToColor(EndpointId endpoint, uint16_t colorX, uin
  * @return Status::Success when successful,
  *         Status::UnsupportedEndpoint when the provided endpoint doesn't correspond with a Color XY transition state (verified in
  * moveToColor function),
+ *         Status::ConstraintError when a command parameters is outside its defined value range.
  */
 Status ColorControlServer::moveToColorCommand(EndpointId endpoint, const Commands::MoveToColor::DecodableType & commandData)
 {
+    // Command Parameters constraint checks:
+    VerifyOrReturnValue(commandData.transitionTime <= kMaxtransitionTime, Status::ConstraintError);
```

Comment:
Why is this not checking colorX and colorY?  Followup OK, but please make sure that happens.

Reply:
moveToColorCommand directly calls a moveToColor that does the check. A few commands have a split handler like this because the method is also used by the scene implementation.

I'm not sure why I added `kMaxtransitionTime` check only here too. I can either remove it or add all constraint checks done in `moveToColor` here too. I think I'll go with the latter.